### PR TITLE
Assets, App Store, and Google Play full content

### PR DESCRIPTION
**Pull request by lisaw-valora**
Full publish of content for MS App Store docs.

### CODE REVIEW

File: docs/publishing/app-store.md [hunk 1]
```diff
@@ -1 +1,86 @@
...
+- Follow the prompts. The process completes when you tap or click the “Subscribe” button
+
+After the purchase, you must wait on Apple for up to 48 hours. **Please watch your email closely for the next steps.**
+
+### Step 1.2: Invite your MS CSM
+
+[Log in to App Store Connect](https://appstoreconnect.apple.com/apps)
+
+- Click “Users & Access”
+- [Invite Valora](https://developer.apple.com/help/account/manage-your-team/invite-team-members/): Invite your MS CSM as an **Admin**
```
Comment (silasbw) [inline docs/publishing/app-store.md:75]: Invite Valora?  Should this be something like `Invite your CSM`?

**Review by silasbw — COMMENTED** [review]
It seems like there's potentially two paths here:
1. some tips on self-serving app/play store setup
2. instructions for adding a MS CSM to help

Is it worth making those two paths distinct?

File: docs/publishing/app-store.md (hunk 1 above)
Comment (lisaw-valora) [reply docs/publishing/app-store.md:75]: The MSM CSM is someone we'll appoint: me, for now.  These docs are to help them see the steps that I'll guide them along.

Comment (lisaw-valora) [reply docs/publishing/app-store.md:75]: > Invite Valora? Should this be something like `Invite your CSM`?

That's a good catch. Yes, it should be.

**Review by silasbw — APPROVED** [review]

**Pull request merged**